board/main.c

```diff
@@ -757,6 +769,9 @@ void handle_spi(uint8_t *data, int len) {
       break;
   }
   spi_tx_dma(spi_tx_buf, 0x44);
```

Comment:
Yes, this is the only place where we need to receive data over SPI, we transmit data in spi_flasher.h:122 but there's no need for this there as we are no waiting for receiving data to be ready.